src/Util/LoggerFactory.php

```diff
@@ -28,7 +28,8 @@ public static function create($channel)
 		$logger = new Monolog\Logger($channel);
 		$logger->pushProcessor(new Monolog\Processor\PsrLogMessageProcessor());
 		$logger->pushProcessor(new Monolog\Processor\ProcessIdProcessor());
-		$logger->pushProcessor(new FriendicaProcessor(LogLevel::DEBUG, 1));
+		$logger->pushProcessor(new Monolog\Processor\UidProcessor());
+		$logger->pushProcessor(new FriendicaProcessor(LogLevel::DEBUG, ['Friendica\Core\Logger']));
```

Comment:
As said, we can change `LogLevel::DEBUG` to any loglevel we want. So the log details won't appear at every log, just above the level we want.
=> Maybe we can decrease the amount of logs in this way